docs/build/get-started/wallet-management.md

```diff
@@ -0,0 +1,190 @@
+# Wallet Management and How to Get the Testnet Tokens
```

Comment:
I think it's a bit too long, so maybe drop "Wallet Management"

<img width="311" alt="image" src="https://github.com/fluencelabs/docs/assets/1949356/1959ef34-529b-482c-87f7-dfc52c707406">


```suggestion
# How to Get the Testnet Tokens
```